magenta-lib/src/test/scala/magenta/input/RiffRaffYamlReaderTest.scala

```diff
@@ -0,0 +1,73 @@
+package magenta.input
+
+import org.scalatest.{FlatSpec, ShouldMatchers}
+import play.api.libs.json.{JsArray, JsNumber, JsString, Json}
+
+class RiffRaffYamlReaderTest extends FlatSpec with ShouldMatchers{
```

Comment:
I think these tests would make any problems with the YAML reading easier to understand so I'm tempted to keep them.